compiler/src/dotty/tools/dotc/consumetasty/ConsumeTasty.scala

```diff
@@ -16,9 +18,23 @@ object ConsumeTasty {
         new TastyFromClass(tastyConsumer)
     }
 
-    val currentClasspath = QuoteDriver.currentClasspath(getClass.getClassLoader)
+    val currentClasspath = getCurrentClasspath(getClass.getClassLoader)
     import java.io.File.{ pathSeparator => sep }
     val args = "-from-tasty" :: "-Yretain-trees" :: "-classpath" :: s"$classpath$sep$currentClasspath" :: classes
     (new Consume).process(args.toArray)
   }
+
+  private def getCurrentClasspath(cl: ClassLoader): String = {
```

Comment:
Why does this need to be duplicated with QuoteDriver ? (Unrelated but this implementation looks very fragile to me, this makes way too many assumptions about how classloaders are setup, which might be broken by sbt 1.3 uses of layered classloaders for example)

Reply:
I will look into it in another PR. For now we keep the same behaviour as before.